nuxt.config.js

```diff
@@ -13,5 +13,13 @@ export default theme({
       ]
     }
   },
+  hooks: {
+    'content:file:beforeInsert': async (document, database) => {
+      document.text = `
+       <alert type="warning">Warning</alert>
+      ${document.text}`;
+      console.log(document); // TODO: do remove this
```

Comment:
I don't see the warning, on the website, but I'm seeing it added in the text.

<img width="1792" alt="Screenshot 2021-02-27 at 2 19 49 PM" src="https://user-images.githubusercontent.com/56079307/109382507-dbd70a80-7906-11eb-8706-f843fa2856ff.png">